eclipse-scout-core/src/desktop/outline/pages/Page.js

```diff
@@ -198,6 +198,14 @@ export default class Page extends TreeNode {
     if (form instanceof Form) {
       form.setModal(false);
       form.setClosable(false);
+
+      form.setDisplayHint(Form.DisplayHint.VIEW);
+      form.displayViewId = 'C';
+
+      form.showOnOpen = false;
+      if (form.rootGroupBox) {
```

Comment:
Why is it necessary to set menubar position explicitly? This prevents having a bottom menubar. I checked some detail forms with Scout Classic and JS, the position is set to AUTO. The GroupBox renders AUTO to top and does not change the value. Does your groupbox have set it to bottom explicitly?

Reply:
The detailForm is created first and therefore _Form.\_init_ is executed before _Page.\_initDetailForm_.
_Form.\_init_ calls _Form.\_setRootGroupBox_ and this sets the _menuBarPosition_ to _'bottom'_ as the default-_displayHint_ of a Form is _'dialog'_. Afterwards _Page.\_initDetailForm_ sets the _displayHint_ of the detailForm to _'view'_.
This only happens for ScoutJS-detailForms that have not set their _displayHint_ to _'view'_, which should not be necessary as the Page handles it.

Possible solutions are:
- Every Form used as detailForm needs to set its _displayHint_ to _'view'_ before it is created
 -> _menuBarPosition = 'bottom'_ is possible but the form can no longer be used somewhere else as a dialog
 -> works for Js-Page with Js-DetailForm and Java-Page with Js-DetailForm
- The page will manipulate its _\_detailFormModel_ before the detailForm is created
 -> _menuBarPosition = 'bottom'_ is possible and the form can be used somewhere else as a dialog but using _Page.setDetailForm_ might give a different result than passing the form in the model
 -> works for Js-Page with Js-DetailForm but not for Java-Page with Js-DetailForm
- Explicitly set the _menuBarPosition_ in _Page.\_initDetailForm_
 -> _menuBarPosition = 'bottom'_ is not possible but the form can be used somewhere else as a dialog and using _Page.setDetailForm_ or passing the form in the model has the same result
 -> works for Js-Page with Js-DetailForm and Java-Page with Js-DetailForm